pageserver/src/walredo.rs

```diff
@@ -165,7 +165,7 @@ impl PostgresRedoManager {
             }
         }
         // last batch
-        if batch_neon {
+        let res = if batch_neon {
             self.apply_batch_neon(key, lsn, img, &records[batch_start..])
```

Comment:
Do you want to set `last_successful_redo_at` if `apply_batch_neon` succeeds? I think no. `apply_batch_neon` doesn't use the walredo process at all.

Reply:
Right! Pushed a fix